charts/nfs-subdir-external-provisioner/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 4.0.2
+appVersion: 4.0.3
 description: nfs-subdir-external-provisioner is an automatic provisioner that used your *already configured* NFS server, automatically creating Persistent Volumes.
 name: nfs-subdir-external-provisioner
 home: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner
-version: 4.0.18
-kubeVersion: ">=1.9.0-0"
+version: 4.0.20
```

Comment:
changing the version will cause chart release, better to change the kubeVersion only when we will actually release the chart with those restrictions, currently it will be redundant since image and chart are same.